script/plugin_webpack_directories.rb

```diff
@@ -27,12 +25,12 @@
 
   path = "#{dep.to_spec.full_gem_path}/webpack"
   entry = "#{path}/index.js"
```

Comment:
Technically I think we should read [main from package.json](https://docs.npmjs.com/files/package.json#main) and use that instead of hardcoding it (with a fallback).

Reply:
That seems easy enough to implement, I'll do it.